jsonify_stored/models/jsonify_stored_mixin.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import json
+from itertools import groupby
+
+from odoo import api, fields, models
+from odoo.tools.misc import split_every
+
+
+class JsonifyStoredMixin(models.AbstractModel):
+    """This mixin aims to provide the ability to store jsonified data on model."""
+
+    _name = "jsonify.stored.mixin"
+
+    def _ir_export_id_domain(self):
+        return [("resource", "=", self._name)]
+
+    # TODO writeable, compute default value, fitler by current model
+    ir_export_id = fields.Many2one(
+        "ir.exports", domain=lambda self: self._ir_export_id_domain()
+    )
+    jsonified_display = fields.Text(
+        compute="_compute_jsonified_display",
+    )
+    jsonified_data = fields.Serialized(compute="_compute_jsonified_data", store=True)
+
+    def _compute_jsonified_display(self):
+        for record in self:
+            record.jsonified_display = json.dumps(
+                record.jsonified_data, sort_keys=True, indent=4
+            )
+
+    def _chunk(self):
```

Comment:
move this to the bottom
```suggestion
    def _chunk(self, size=1000):
```